### PR TITLE
Add documentation for new Enphase Envoy platform

### PR DESCRIPTION
**Pull request by jesserizzo**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#15081

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Looks great @jesserizzo!
I did remove the configuration header/title thingy, since that is already handled by our configuration tags.

✅ Approved.

Can be merged as soon as the parent PR gets merged.

File: source/_components/enphase_envoy.markdown [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+title: "Enphase Envoy"
+description: "Instructions on how to setup Enphase Envoy with Home Assistant."
+date: 2018-06-30 12:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: enphase.svg
+ha_category: Energy
+ha_release: 0.73
```
Comment (MartinHjelmare) [inline source/_components/enphase_envoy.markdown]: 0.76

File: source/_components/enphase_envoy.markdown [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+ip:
+  description: The local IP Address of your Envoy
+  required: true
+  type: string
+name:
+  description: Name to use in the front end
+  required: false
+  type: string
+monitored_conditions:
+  description: The list of conditions to display in the front end
```
Comment (MartinHjelmare) [inline source/_components/enphase_envoy.markdown]: This doesn't have anything to do with the frontend per se. This decides what entities will be created, ie what data to collect.

File: source/_components/enphase_envoy.markdown [hunk 3]
```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title: "Enphase Envoy"
+description: "Instructions on how to setup Enphase Envoy with Home Assistant."
+date: 2018-06-30 12:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: enphase.svg
```
Comment (MartinHjelmare) [inline source/_components/enphase_envoy.markdown]: The filename here doesn't match the included file.

File: source/_components/enphase_envoy.markdown [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  - platform: enphase_envoy
+    ip: ENVOY_LOCAL_IP_ADDRESS
+```
+
+{% configuration %}
+ip:
+  description: The local IP Address of your Envoy
+  required: true
+  type: string
+name:
```
Comment (MartinHjelmare) [inline source/_components/enphase_envoy.markdown]: This isn't included in the config schema of the platform.

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]
The name of the markdown file is incorrect. It should be prefixed with `sensor.`.

File: source/_components/sensor.enphase_envoy.markdown [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  - platform: enphase_envoy
+    ip_address: ENVOY_LOCAL_IP_ADDRESS
+```
+
+{% configuration %}
+ip_address:
+  description: The local IP Address of your Envoy
+  required: true
+  type: string
+name:
```
Comment (MartinHjelmare) [inline source/_components/sensor.enphase_envoy.markdown]: Remove this key.

**Review by MartinHjelmare — APPROVED** [review]
Great!

**Pull request merged**